.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  presets: [
+    'es2015',
+    'es2016',
+    'react'
+  ],
+  plugins: [
+    'transform-class-properties',
```

Comment:
Please explain what each of these transforms do - I'm not convinced we need any of them?